dvc/command/diff.py

```diff
@@ -11,6 +11,21 @@
 logger = logging.getLogger(__name__)
 
 
+def _show_md(diff):
+    from dvc.utils.diff import table
+
+    rows = []
+    for status in ["added", "deleted", "modified"]:
+        entries = diff.get(status, [])
+        if not entries:
+            continue
+        paths = sorted([entry["path"] for entry in entries])
+        for path in paths:
+            rows.append([status, path])
+
+    return table(["Status", "Path"], rows, True)
```

Comment:
Should the output's column be swapped?


| Path   | Status   |
|--------|------------|
| model.pkl | Added |

Reply:
^ and also capitalized?

Reply:
> Should the output's column be swapped?

Currently, it is in the order in which it was requested. And, which matches `git status`.

> ^ and also capitalized?

not capitalized because it was requested in that form and matches `git status`.